.github/workflows/lint.yml

```diff
@@ -24,6 +24,6 @@ jobs:
 
             - run: npm ci
 
-            - run: npm run lint
+            - run: npm run lint -- --max-warnings=0
```

Comment:
is this redundant if you added the same flag to the npm script?

Reply:
Oops, let me check.

Reply:
Yes, [looks redundant](https://github.com/Expensify/Expensify.cash/pull/1611/checks?check_run_id=2023974366#step:6:8), let me remove this one